.github/ISSUE_TEMPLATE/bug_report_en.yml

```diff
@@ -0,0 +1,57 @@
+name: 🐞 Bug Report Form
+description: English template for reporting a bug
+title: "[🐞 Report]: "
+labels: [ "triage" ]
+
+body:
+  - type: markdown
+    attributes:
+      value: >-
+        Before reporting an error, you'd better contact A248 on [Discord](https://discord.gg/3C4qeG8XhE) for support.
+        Please ensure that there is no similar existing one reported that.
+        <br>Please fill in the following content completely.
+  - type: input
+    id: version
+    attributes:
+      label: LibertyBans Version
+      description: Run `/libertybans version` to get it. Latest version may get on [Jenkins CI](https://jenkins.hahota.net/job/LibertyBans/)
+      placeholder: LibertyBans 0.8
+    validations:
+      required: true
+      
+ 
+  - type: checkboxes
+    id: preliminary
+    attributes:
+      label: I have confirmed that ...
+      options:
+        - label: LibertyBans is up to date
+        - label: There is no similar issue that has been reported
+          required: true
+        - label: I have read Wiki and ensure this problem not related to my actions.
+          required: true
+      
+  - type: dropdown
+    attributes:
+      label: Platform
+      options:
+        - "Spigot"
+        - "Paper"
+        - "Bungeecord"
+        - "Velocity"
+      multiple: true
+    validations:
+      required: true
+
+  - type: textarea
+    id: description
+    attributes:
+      label: Description
+      description: >-
+        Please include as much information as possible. 
+        <br>For the description,
+        be as detailed as possible and include a step by step reproduction. 
+        <br>The clearer the description, the higher the report processing priority
```

Comment:
```suggestion
```
It should be relatively obvious why a clearly described bug report is better than a lacking one. I'm satisfied to trust users in understanding these reasons.